plugins/env/app/views/samson_env/_deploy_tab_body.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="tab-pane" id="environment-variables">
+  <%
+    previous_env = @deploy.previous_successful_deploy&.env_state
+    current_env = @deploy.persisted? ? @deploy.env_state : @deploy.serialized_environment_variables
```

Comment:
semi bad since it replicates internal deploy logic (when state is stored + where it is stored from)